### PR TITLE
Feature/345 inversion solution nrml objects

### PR DESCRIPTION
**Pull request by 0qy**

### CODE REVIEW

**Review by benjamineac — APPROVED** [review]

**Review by chrisdicaprio — APPROVED** [review]

**Review by chrisdicaprio — CHANGES_REQUESTED** [review]
@0qy the InversionSolution ID as a navlink dosn't change depending on type (`ScaledInversionSolution` vs `InversionSolution`). Results in ID not found if type is `ScaledInversionSolution`

**Comment by chrisbc** [comment]
> @0qy the InversionSolution ID as a navlink dosn't change depending on type (`ScaledInversionSolution` vs `InversionSolution`). Results in ID not found if type is `ScaledInversionSolution`

@0qy and I looked at this and the problem is the data . the InvSol ID has been saved in API instead of the ScaledIS. See [new Runzi ticket](https://github.com/GNS-Science/nzshm-runzi/issues/103)

**Comment by chrisbc** [comment]
@chrisdicaprio can you give your approval on this one pls? (following our discussion re runzi data / ticket )

File: src/pages/InversionSolutionNrml.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                sourceSolutionType === UnifiedInversionSolutionType.INVERSION_SOLUTION
+                  ? `/InversionSolution/${data?.node?.source_solution?.id}`
+                  : `/ScaledInversionSolution/${data?.node?.source_solution?.id}`
+              }
+            >
+              {data?.node?.source_solution?.id}
+            </a>
+          </Typography>
+          <Typography>
+            <a href={data?.node?.file_url ?? ''}>Downlaod</a>
```
Comment (chrisdicaprio) [inline src/pages/InversionSolutionNrml.tsx:61]: typo "Download"

**Review by chrisdicaprio — CHANGES_REQUESTED** [review]
fix typo

**Review by chrisdicaprio — APPROVED** [review]

**Pull request merged**